<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="7" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="3790" started-at="2016-03-08T19:46:02Z" finished-at="2016-03-08T19:46:06Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="3790" started-at="2016-03-08T19:46:02Z" finished-at="2016-03-08T19:46:06Z">
      <class name="com.redhat.osas.hibernate.util.SessionBuilderTest">
        <test-method status="PASS" signature="testSessionFactory()[pri:0, instance:com.redhat.osas.hibernate.util.SessionBuilderTest@2d6e8792]" name="testSessionFactory" duration-ms="767" started-at="2016-03-08T14:46:05Z" finished-at="2016-03-08T14:46:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSessionFactory -->
      </class> <!-- com.redhat.osas.hibernate.util.SessionBuilderTest -->
      <class name="com.redhat.osas.jpa.util.JPASessionUtilTest">
        <test-method status="PASS" signature="getEntityManager()[pri:0, instance:com.redhat.osas.jpa.util.JPASessionUtilTest@22f71333]" name="getEntityManager" duration-ms="2350" started-at="2016-03-08T14:46:02Z" finished-at="2016-03-08T14:46:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEntityManager -->
        <test-method status="PASS" signature="getSession()[pri:0, instance:com.redhat.osas.jpa.util.JPASessionUtilTest@22f71333]" name="getSession" duration-ms="1" started-at="2016-03-08T14:46:05Z" finished-at="2016-03-08T14:46:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getSession -->
        <test-method status="PASS" signature="nonexistentEntityManagerName()[pri:0, instance:com.redhat.osas.jpa.util.JPASessionUtilTest@22f71333]" name="nonexistentEntityManagerName" duration-ms="60" started-at="2016-03-08T14:46:05Z" finished-at="2016-03-08T14:46:05Z">
          <exception class="javax.persistence.PersistenceException">
            <message>
              <![CDATA[No Persistence provider for EntityManager named nonexistent]]>
            </message>
            <full-stacktrace>
              <![CDATA[javax.persistence.PersistenceException: No Persistence provider for EntityManager named nonexistent
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:61)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at com.redhat.osas.jpa.util.JPASessionUtil.getEntityManager(JPASessionUtil.java:17)
	at com.redhat.osas.jpa.util.JPASessionUtilTest.nonexistentEntityManagerName(JPASessionUtilTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- javax.persistence.PersistenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- nonexistentEntityManagerName -->
        <test-method status="PASS" signature="nonexistentSessionName()[pri:0, instance:com.redhat.osas.jpa.util.JPASessionUtilTest@22f71333]" name="nonexistentSessionName" duration-ms="31" started-at="2016-03-08T14:46:05Z" finished-at="2016-03-08T14:46:05Z">
          <exception class="javax.persistence.PersistenceException">
            <message>
              <![CDATA[No Persistence provider for EntityManager named nonexistent]]>
            </message>
            <full-stacktrace>
              <![CDATA[javax.persistence.PersistenceException: No Persistence provider for EntityManager named nonexistent
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:61)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at com.redhat.osas.jpa.util.JPASessionUtil.getEntityManager(JPASessionUtil.java:17)
	at com.redhat.osas.jpa.util.JPASessionUtil.getSession(JPASessionUtil.java:24)
	at com.redhat.osas.jpa.util.JPASessionUtilTest.nonexistentSessionName(JPASessionUtilTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- javax.persistence.PersistenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- nonexistentSessionName -->
        <test-method status="PASS" signature="testEntityManager()[pri:0, instance:com.redhat.osas.jpa.util.JPASessionUtilTest@22f71333]" name="testEntityManager" duration-ms="510" started-at="2016-03-08T14:46:05Z" finished-at="2016-03-08T14:46:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEntityManager -->
        <test-method status="PASS" signature="testSession()[pri:0, instance:com.redhat.osas.jpa.util.JPASessionUtilTest@22f71333]" name="testSession" duration-ms="26" started-at="2016-03-08T14:46:05Z" finished-at="2016-03-08T14:46:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSession -->
      </class> <!-- com.redhat.osas.jpa.util.JPASessionUtilTest -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
